content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+membutuhkan perintah ini untuk [menggabungkan simpul-simpul ke klaster](#menggabungkan-simpul).
+
+_Token_ digunakan untuk otentikasi mutual antara simpul _control-plane_ dan simpul-simpul yang
+akan bergabung. _Token_ yang didapat di sini bersifat rahasia. Simpan dengan aman, karena siapapun yang memiliki token tersebut
+dapat menambahkan simpul-simpul yang otentik ke klaster. Kamu dapat menampilkan daftar _token_,
+membuat, dan menghapus _token_ dengan perintah `kubeadm token`. Lihat
+[panduan referensi kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm-token/).
+
+### Memasang _add-on_ jaringan Pod {#jaringan-pod}
+
+{{< caution >}}
+Bagian ini berisi informasi penting mengenai penyetelan jejaring dan
+urutan penyebaran.
+Baca seluruh saran ini dengan saksama sebelum melanjutkan.
+
+**Kamu harus menyebarkan
+_add-on_ jaringan Pod berbasis {{< glossary_tooltip text="_Container Network Interface_" term_id="cni" >}}
+(CNI) sehingga Pod dapat berkomunikasi satu sama lain.  
+DNS klaster (CoreDNS) tidak akan menyala sebelum jaringan dipasangkan.**
+
+- Perlu diperhatikan bahwa jaringan Pod tidak boleh tumpang tindih dengan jaringan _host_
+  manapun: kamu akan menemui beberapa masalah jika terjadi tumpang tindih.  
+  (Jika kamu menemukan adanya bentrokan antara jaringan Pod
+  pilihan _plugin_ jaringan dengan jaringan _host_, kamu harus memikirkan blok
+  CIDR yang cocok untuk digunakan, kemudian menggunakannya pada saat `kubeadm init` dengan
+  `--pod-network-cidr` dan sebagai pengganti pada YAML _plugin_ jaringan).
+
+- Secara bawaan, `kubeadm` mengatur klastermu untuk menggunakan dan melaksanakan penggunaan
+  [RBAC](/docs/reference/access-authn-authz/rbac/) (role based access
+  control).  
+  Pastikan _plugin_ jaringan Pod mendukung RBAC, dan begitu juga seluruh manifes
+  yang kamu gunakan untuk menyebarkannya.
+
+- Jika kamu ingin menggunakan IPv6--baik jaringan _dual-stack_, ataupun jaringan _single-stack_ IPv6
+  --untuk klastermu, pastikan _plugin_ jaringan Pod
+  mendukung IPv6.  
+  Dukungan IPv6 telah ditambahkan pada CNI sejak [v0.6.0](https://github.com/containernetworking/cni/releases/tag/v0.6.0).
+
+{{< /caution >}}
+
+{{< note >}}
+Saat ini Calico adalah satu-satunya _plugin_ CNI yang dapat menerima uji e2e (_end-to-end_) oleh proyek kubeadm.
+Jika kamu menemukan isu terkait _plugin_ CNI kamu harus membuat tiket pada pelacak isu masing-masing _plugin_
+alih-alih pada pelacak isu kubeadm maupun kubernetes.
+{{< /note >}}
+
+Beberapa proyek eksternal menyediakan jaringan Pod Kubernetes menggunakan CNI, beberapa di antaranya juga
+mendukung [Network Policy](/docs/concepts/services-networking/networkpolicies/).
+
+Lihat daftar
+[_add-on_ jejaring dan _network policy_](https://kubernetes.io/docs/concepts/cluster-administration/addons/#networking-and-network-policy) yang tersedia.
+
+Kamu dapat memasang _add-on_ jaringan Pod dengan perintah berikut pada simpul
```

Comment:
```suggestion
Kamu dapat memasang _add-on_ jaringan Pod dengan perintah berikut pada Node
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc